src/services/rns/rns.processor.ts

```diff
@@ -37,25 +33,28 @@ async function transferHandler (logger: Logger, eventData: EventData): Promise<v
     id: transactionHash,
     tokenId,
     sellerAddress: from,
-    newOwnerAddress: ownerAddress
+    buyerAddress: ownerAddress
   })
 
   if (transferDomain) {
     logger.info(`Transfer event: Transfer ${tokenId} created`)
   }
+
+  const domainsService = services.domains
   const domain = await Domain.findByPk(tokenId)
 
   if (domain) {
     await DomainOwner.upsert({ address: ownerAddress, tokenId })
+    await domainsService.patch(domain.tokenId, { tokenId }) // Trigger patch event
```

Comment:
Wait... 😳 So you are updating here `tokenId` to the same value, just to trigger the event?

Did you try to use simple `domainsService.emit('patched', <data>)` instead?

Reply:
Not sure why I did that, lol.

Reply:
Ah no I do know. because we don't always have access to the emit object. but I guess I could just check instead of writing to db unnecessarily